wlc/1Kgs.xml

```diff
@@ -6542,7 +6542,7 @@
           <w lemma="3967" morph="HAcbpa" id="11jTj">מֵא֣וֹת</w>
           <w lemma="3701" n="1.1" morph="HNcmsa" id="11yoM">כֶּ֔סֶף</w>
           <w lemma="c/5483 b" n="1.0" morph="HC/Ncmsa" id="11u9c">וְ/ס֖וּס</w>
-          <w lemma="b/2572" morph="HRd/Acbpa" id="11npB">בַּ/חֲמִשִּׁ֣ים</w>
+          <w lemma="b/2572" morph="HR/Acbpa" id="11npB">בַּ/חֲמִשִּׁ֣ים</w>
```

Comment:
Patah vowel already explained by if the noun begins with a guttural with a hatep vowel, the preposition takes the corresponding full vowel

Reply:
Yes, I thought that was the logic :+1: So I've seen more of these in the same pull request, did you systematically check for cases of a number in the hundreds and whether the direct article was tagged or not?

Reply:
A possible reference in support is JM §35.e  which mentions a similar case where it cannot be known whether the definite article is present or not. It would indeed be safer in that case not to assume the article is there; this is also supported by other parallels.

Reply:
Yes, Joel & I actually did use JM §35.e in support as we wrestled through each instance on a case-by-case basis.